plugins/jobtype/src/azkaban/jobtype/HadoopSparkJob.java

```diff
@@ -312,7 +370,8 @@ private static void handleStandardArgument(Props jobProps,
 
   @Override
   protected List<String> getClassPaths() {
-
+    // The classpath for the process that runs HadoopSecureSparkWrapper
+    String pluginDir = getSysProps().get("plugin.dir");
```

Comment:
I used this var somewhere else but later removed the code. This is not a necessary change